gulpfile.js

```diff
@@ -107,6 +109,12 @@ gulp.task('copy:videos', 'copy example videos', function() {
       .pipe(gulp.dest(paths.dist.video));
 });
 
+gulp.task('copy:jsons', 'copy example jsons', function() {
```

Comment:
nit: there is no plural of sons (JavaScript Object Notation). Suggestion: rename to copy:json.